docs/src/explanations/chisel-enum.md

```diff
@@ -0,0 +1,130 @@
+# ChiselEnum
+
+The ChiselEnum type can be used to reduce the chance of error when encoding mux selectors, opcodes, and functional unit operations. In contrast with`Chisel.util.Enum`, `ChiselEnum` are subclasses of `Data`, which means that they can be used to define fields in `Bundle`s, including in `IO`s.
+
+
+## Functionality and Examples
+
+```scala mdoc
+// Imports used in the following examples
+import chisel3._
+import chisel3.util._
+import chisel3.stage.ChiselStage
+import chisel3.experimental.ChiselEnum
```

Comment:
```suggestion
import chisel3.experimental.ChiselEnum
```

```

Reply:
sigh. i swear i am suggesting the additional backticks!

Reply:
I pushed a new commit with backticks